go/vt/sidecardb/sidecardb.go

```diff
@@ -368,7 +368,11 @@ func (si *schemaInit) setCurrentDatabase(dbName string) error {
 func (si *schemaInit) getCurrentSchema(tableName string) (string, error) {
 	var currentTableSchema string
 
-	rs, err := si.exec(si.ctx, sqlparser.BuildParsedQuery(showCreateTableQuery, GetIdentifier(), sqlparser.String(sqlparser.NewIdentifierCS(tableName))).Query, 1, false)
+	// We escape the tableName because it can be a keyword.
+	// Converting the tableName to a case-sensitive identifier and printing it out using the
```

Comment:
sorry to be nit-picky, but we aren't "printing it out".
I'll approve anyway, maybe you can re-word it to just remove the "and printing it out", it's not necessary.

Reply:
Fixed.